docs/releases/2.0.txt

```diff
@@ -305,6 +301,11 @@ Miscellaneous
 * :class:`~django.views.generic.base.RedirectView` no longer silences
   ``NoReverseMatch`` if the ``pattern_name`` doesn't exist.
 
+* :class:`~django.forms.FloatField` and :class:`~django.forms.DecimalField`
+  now respect :setting:`DECIMAL_SEPARATOR` and :setting:`THOUSAND_SEPARATOR`
+  during validation correctly when :setting:`USE_L10N` is off.
```

Comment:
I'm not sure if the note describes the issue in enough detail. (Why is it considered backwards incompatible?) Perhaps giving an example would help? Also "correctly" seems out of place in the sentence. \\cc @claudep